downstream/modules/platform/ref-controller-image-verification.adoc

```diff
@@ -0,0 +1,12 @@
+[id="ref-controller-image-verification"]
+
+= image verification
```

Comment:
Should this title be capitalised? I can see why other headings that are variables / keywords etc are lowercase, but is "image verification" a conceptual thing rather than a setting in code?